commitlint.config.js

```diff
@@ -1 +1,4 @@
-module.exports = { extends: ["@commitlint/config-conventional"] }
+module.exports = {
+  extends: ["@commitlint/config-conventional"],
+  rules: { "body-max-line-length": [0, "always"] },
```

Comment:
This is fine for now. In the long run, a solution closer to the cause of the problem will be better. From my understanding, the excessive line length is caused by dependabot's verbose commit messages (that no one reads? 😉) containing links to package sources, etc. Maybe there's a way to make dependabot a bit more tight-lipped and have it reduce its line length to a value we define 🤷

Reply:
I don't think that's the problem. I think the release commit generates a description of the release and puts it in the body. Here's it

```
Command failed with exit code 1: git commit -m chore(release): 1.1.1 [skip ci]\n' +
    '\n' +
    '## [1.1.1](https://github.com/foobaragency/cf-migrations/compare/v1.1.0...v1.1.1) (2021-07-23)\n' +
    '\n' +
    '\n' +
    '### Bug Fixes\n' +
    '\n' +
    "* don't save migrations that were aborted ([7352809](https://github.com/foobaragency/cf-migrations/commit/73528099e7ae087fd6d0900c02202cb17a45bf97))\n" +
    '* duplicate entry in migrationResults ([e18d159](https://github.com/foobaragency/cf-migrations/commit/e18d1598b63f495e5a8d5f5e628515c6dcff77b4))\n' +
    '\n' +
    '\n' +
    '\n',
```

That's the commit that got rejected (AFAIK).

Reply:
I understood that the commit message causing issues is the one generated by semantic-version itself.